mpt_extension_sdk/swo_rql/query_builder.py

```
@@ -34,22 +37,48 @@ def parse_kwargs(query_dict):
+    return value
+
+
+def rql_encode(op: str, value):
+    """Encode a Python value to its RQL string representation for a given operator."""
+    if op in constants.LIST:
+        if isinstance(value, list | tuple):
+            encoded_items = []
+            for item in value:
+                encoded = _encode_scalar(item)
+                if encoded is None:
+                    raise TypeError(f"the `{op}` operator doesn't support the {type(item)} type.")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)